include/sys/fs/zfs.h

```diff
@@ -756,6 +762,15 @@ typedef enum pool_state {
 } pool_state_t;
 
 /*
+ * mmp state. The following states provide additional detail describing
+ * why a pool couldn't be safely imported.
+ */
+typedef enum mmp_state {
+	MMP_STATE_ACTIVE = 0,		/* In active use		*/
+	MMP_STATE_NO_HOSTID		/* System hostid is not set	*/
+} mmp_state_t;
```

Comment:
Will `ZPOOL_CONFIG_IMPORT_STATE` in the pool config only represent some form of mmp state? I feel like `ZPOOL_CONFIG_IMPORT_STATE` could be useful in the future as a generic method to indicate why import failed. If `ZPOOL_CONFIG_IMPORT_STATE` is only ever going to be used by mmp, then the key should be named `ZPOOL_CONFIG_MMP_STATE` instead? Otherwise, if `ZPOOL_CONFIG_IMPORT_STATE` was intended to be generic, we should rename the enum to be generic as well.